tools/scaffold/scripts/schema-scaffold.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+
```

Comment:
the script should fail here in case `name` is not defined.

Reply:
makes sense, adjusted